src/platform/i386/exception.c

```diff
@@ -11,6 +11,7 @@
 void
 print_regs_state(struct pt_regs *regs)
 {
+/*
```

Comment:
Why remove these?

Reply:
> Why remove these?

I remove this part because I use qemu for the test, when the device not available interruption is triggered, the print out of this part will have bad influence on the result, e.g. thread cannot switch. I will undo this removal.

Reply:
Thanks for the context here. I think that we want it uncommented (as with your updated PR), but you can **remove** the call to print out the registers in the co-processor exception handler. The goal is that 1. the system has to run correctly (so we can't have the  prints in the co-processor exception), and 2. we need to print out the registers in other fault handlers to help debug. So we activate the register print outs in all other exception handlers, but not in yours.

Reply:
Thank you for your comments, I've (1) undone the removal of the print out, (2) removed the calling of print_regs_state() for device not available exception.